src/contracts/pods/EigenPodManager.sol

```diff
@@ -145,14 +145,14 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
     /**
      * @notice Removes beacon chain ETH from EigenLayer on behalf of the owner of an EigenPod, when the
      *         balance of a validator is lower than how much stake they have committed to EigenLayer
-     * @param podOwner The owner of the pod whose balance must be removed.
-     * @param beaconChainETHStrategyIndex is the index of the beaconChainETHStrategy for the pod owner for the callback to 
-     *                                    the StrategyManager in case it must be removed from the list of the podOwner's strategies
-     * @param amount The amount of beacon chain ETH to decrement from the podOwner's shares in the strategyManager.
+     * @param podOwner is the pod owner to be slashed
```

Comment:
I commented this elsewhere but the podOwner isn't getting 'slashed' here, so the comment is definitely wrong